base/multidimensional.jl

```diff
@@ -1565,8 +1565,8 @@ end
     end
 end
 
-isassigned(a::AbstractArray, i::CartesianIndex) = isassigned(a, Tuple(i)...)
-function isassigned(A::AbstractArray, i::Union{Integer, CartesianIndex}...)
+@propagate_inbounds isassigned(a::AbstractArray, i::CartesianIndex) = isassigned(a, Tuple(i)...)
+@propagate_inbounds function isassigned(A::AbstractArray, i::Union{Integer, CartesianIndex}...)
     isa(i, Tuple{Vararg{Int}}) || return isassigned(A, CartesianIndex(to_indices(A, i)))
```

Comment:
```suggestion
@propagate_inbounds isassigned(A::AbstractArray, i::Union{Integer, CartesianIndex}...) = isassigned(A, to_indices(A, i)...)
@inline function isassigned(A::AbstractArray, I::Int...)
```

Reply:
I've updated the code to something similar (`Integer` instead of `Int` arguments to avoid ambiguities).